modules/json/src/json-converter.js

```diff
@@ -125,6 +129,24 @@ function convertClassInstance(json, configuration) {
   return instantiateClass(type, props, configuration);
 }
 
+// Plain JS object, embed functions.
+function convertFunctionObject(json, configuration) {
+  if (configuration.functions && configuration.convertFunction) {
+    const {functionKey} = configuration;
+
+    const targetFunction = json[functionKey];
+    const dataProp = json.prop;
+    const availableFunctions = configuration.functions;
+
+    const fn = availableFunctions[targetFunction];
+    const wrapperFn = fn(json);
+
+    return configuration.convertFunction(dataProp, configuration, wrapperFn);
```

Comment:
This is a very strange pattern for a user-facing configuration. Why does a user need to deal with `wrapperFn` if they are to provide their own expression evaluator? It can easily be hidden with:

```js
  return row => wrapperFn(configuration.convertFunction(dataProp, configuration));
```